dcos/package.py

```diff
@@ -845,7 +846,15 @@ def copy_to_cache(self, target_dir):
                 tmp_file = os.path.join(tmp_dir, 'packages.zip')
 
                 # Download the zip file.
-                urllib.request.urlretrieve(self.url, tmp_file)
+                requests.packages.urllib3.disable_warnings()
```

Comment:
We have a helper method for this: https://github.com/mesosphere/dcos-cli/blob/master/dcos/http.py#L195-L197.